lib/src/repository/traits/power_repository.dart

```diff
@@ -1,9 +1,23 @@
 import 'package:gql_link/gql_link.dart';
+import 'package:yonomi_platform_sdk/src/queries/devices/get_device/query.data.gql.dart';
+import 'package:yonomi_platform_sdk/src/queries/devices/get_devices/query.data.gql.dart';
 import 'package:yonomi_platform_sdk/src/queries/power/make_power_action_request/query.req.gql.dart';
 import 'package:yonomi_platform_sdk/src/repository/repository.dart';
 import 'package:yonomi_platform_sdk/yonomi-sdk.dart';
 
 class PowerRepository {
+  static PowerTrait getPowerTrait(dynamic trait) {
+    if (trait is GgetDeviceData_device_traits__asPowerDeviceTrait ||
+        trait
+            is GgetDevicesData_me_devices_edges_node_traits__asPowerDeviceTrait) {
+      return PowerTrait(IsOnOff(trait.state.isOn.reported?.value ?? false),
```

Comment:
Just noticed this as well - I don't think we should default the power state to off in the event that it's null.  Let's just allow it to be null and handle it.  I'm unconvinced that we should ever be putting in a default value when the API returns null.